docs/insomnia/get-started-with-documents.md

```diff
@@ -0,0 +1,122 @@
+---
+layout: article-detail
+title: Get Started with Documents
+category: "API Design"
+category-url: api-design
+---
+
+Design Documents (also called Documents) hold specifications, API requests, and tests. They also offer the capability to sync with Git.
+
+If you're looking for information about Collections instead (which use Version Control Sync), see [Request Collections](/insomnia/request-collections).
+
+For this guide, create an empty Document from your Dashboard. We've named our Document "petstore" as our example uses the Pet Store API.
+
+## Tab Overview
+
+We're going to start in the Design tab. Then we'll walk through each tab in order from left to right.
```

Comment:
simplify to "We'll walk through each tab in order from left to right, starting with the Design tab."